plugins/gradle/src/org/jetbrains/plugins/gradle/statistics/GradleSettingsCollector.kt

```diff
@@ -144,6 +147,7 @@ internal class GradleSettingsCollector : ProjectUsagesCollector() {
                                                           EventFields.Enum("value", TestRunner::class.java) { it.name.lowercase() })
   private val GRADLE_JVM_TYPE = GROUP.registerEvent("gradleJvmType", JRE_TYPE_FIELD)
   private val GRADLE_JVM_VERSION = GROUP.registerEvent("gradleJvmVersion", VERSION_FIELD)
+  private val GRADLE_DAEMON_JVM_CRITERIA_DEFINED = GROUP.registerEvent("gradleDaemonJvmCriteriaDefined", EventFields.Enabled)
```

Comment:
Could you please increase the `GradleSettingsCollector#GROUP` version? 

Our statistics team need to review and create a new schema for this new event.